README.md

```diff
@@ -454,6 +454,37 @@ AS confederation parameters. Valid values are String, keyword 'default'.
 ##### `enforce_first_as`
 Enable/Disable enforces the neighbor autonomous system to be the first AS number listed in the AS path attribute for eBGP. Valid values are 'true', 'false', and 'default'.
 
+##### `event_history_cli`
+Enable/Disable cli event history buffer. Valid values are 'true', 'false', and 'default'.
+
+##### `event_history_cli_size`
+Set cli event history buffer size. Valid values are 'small', 'medium', 'large', 'disable' or 'default'.
```

Comment:
Sorry, not sure I understand your response. What does it mean as an operational state for the buffer to be enabled (`event_history_cli == true`) but the buffer size is disabled (`event_history_cli_size == disable`)? That doesn't intuitively make any sense to me.